cbpro/authenticated_client.py

```diff
@@ -870,13 +870,16 @@ def coinbase_withdraw(self, amount, currency, coinbase_account_id):
         return self._send_message('post', '/withdrawals/coinbase-account',
                                   data=json.dumps(params))
 
-    def crypto_withdraw(self, amount, currency, crypto_address):
+    def crypto_withdraw(self, amount, currency, crypto_address, destination_tag, no_destination_tag=True):
```

Comment:
We should have a default argument for destination_tag as well, for currencies that do not support them.

Reply:
REBEL